### PR TITLE
4.0.7 release

### PR DESCRIPTION
**Pull request by burtbeckwith**

### CODE REVIEW

**Review by Julia3072 — APPROVED** [review]

File: gcn/micronautVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-4.0.3
\ No newline at end of file
+4.0.6
```
Comment (ezzarghili) [inline gcn/micronautVersion.txt:1]: Is this the correct version ? should this be 4.0.7

File: gcn/gcn-core/src/main/java/cloud/graal/gcn/GcnVersionInfo.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            }
+        }
+    }
+
+    public static String getMicronautVersion() {
+        Object micronautVersion = VERSIONS.get("micronaut.platform.version");
+        if (micronautVersion != null) {
+            return micronautVersion.toString();
+        }
+        return "2.0.0";
```
Comment (ezzarghili) [inline gcn/gcn-core/src/main/java/cloud/graal/gcn/GcnVersionInfo.java:46]: Does not look correct.

**Review by ezzarghili — CHANGES_REQUESTED** [review]

File: gcn/micronautVersion.txt (hunk 1 above)
Comment (burtbeckwith) [reply gcn/micronautVersion.txt:1]: this is the Micronaut "core" version, the "platform" version is in  https://github.com/oracle/gcn/blob/main/gcn/gcn-core/src/main/resources/gcn-platform-versions.properties

File: gcn/gcn-core/src/main/java/cloud/graal/gcn/GcnVersionInfo.java (hunk 2 above)
Comment (burtbeckwith) [reply gcn/gcn-core/src/main/java/cloud/graal/gcn/GcnVersionInfo.java:46]: this is just copied from the Micronaut class. There is no sensible default, but the version will be read from the properties file

**Pull request merged**